script/copy_deduplicate

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env python3
 
 """
-Copy a day's data from live ping tables to stable ping tables,
-deduplicating on document_id.
+Copy a day's data from live to stable ping tables, deduplicating on document_id.
```

Comment:
I had to reword a few docstrings now that pydocstyle is working again, because the sentence must be a single line.